pkg/splunk/enterprise/indexercluster.go

```diff
@@ -414,6 +425,39 @@ func (mgr *indexerClusterPodManager) getClusterMasterClient() *splclient.SplunkC
 	return mgr.newSplunkClient(fmt.Sprintf("https://%s:8089", fqdnName), "admin", adminPwd)
 }
 
+func getSiteRepFactor(siteRepFactor string) int32 {
+	re := regexp.MustCompile(".*origin:(?P<rf>.*),.*")
```

Comment:
I am checking for `origin` instead of `total` here because each CR represents number of peers in each site and we should have minimum of `site_replication_factor= origin:` number of peers in each site.